hs-bindgen/fixtures/documentation/doxygen_docs/Example.hs

```diff
@@ -606,6 +608,8 @@ pattern STATUS_ERROR = Status_code_t (-99)
 
   Allows access to high and low parts separately
 
+__C declaration:__ @struct \@data_union_t_as_parts@
```

Comment:
This too is a true change, but a good one: we now record the anonymous C declaration, using the same syntax as in binding specs.